libs/akita/src/lib/entityStore.ts

```diff
@@ -512,95 +534,106 @@ export class EntityStore<S extends EntityState = any, EntityType = getEntityType
    *
    * }
    */
-  createUIStore(initialState = {}, storeConfig: Partial<StoreConfigOptions> = {}) {
+  createUIStore(initialState = {}, storeConfig: Partial<StoreConfigOptions> = {}): EntityUIStore<any, EntityType> {
     const defaults: Partial<StoreConfigOptions> = { name: `UI/${this.storeName}`, idKey: this.idKey };
+    // TODO EntityUIStore uses EntityStore, and EntityStore uses EntityUIStore. This is bad, fix!
```

Comment:
?

Reply:
The file have 2 classes defined and I was unable to move them into their own files as they're using each other. So I had to `/** eslint-disable max-classes-per-file */` and this TODO serves as a reminder to do something about it